src/components/Home/ClusterDashboardItem.js

```diff
@@ -251,6 +254,13 @@ function ClusterDashboardItem({
               </span>
             </RefreshableLabel>
             <Dot style={{ paddingLeft: 0 }} />
+            <RefreshableLabel value={release.kubernetesVersion}>
```

Comment:
The problem is possibly because if the release version is not found, then the value of `release` is `null`, which doesn't have a `kubernetesVersion` property.

This should be enough here.

```suggestion
            <RefreshableLabel value={release?.kubernetesVersion}>
```

Reply:
Done